trunner/testcase.py

```diff
@@ -261,11 +262,12 @@ def handle(self, proc):
         return res
 
 
-class TestCaseBusybox(TestCase):
-    """The test case representing Busybox test suite."""
+class TestCaseLong(TestCase):
```

Comment:
If I'm looking right, `TestCaseLong` and `TestCaseUnit` are the same, maybe we should share to code by a common class?

Reply:
After switching to `3rdpartyTestHarnessFactory` it can be combined into one.